src/error.ts

```diff
@@ -55,7 +55,7 @@ export class DiagnosticPrinter {
 
     const path = this._formatPath(diagnostic.location);
     let result = `❌ ${path} ${diagnostic.message}
-${drawSquiggle(diagnostic.location, 4)}`;
+    ${drawSquiggle(diagnostic.location, 4)}`;
```

Comment:
I think the previous indentation was intentional, because we don't want to print that indentation on the terminal when this diagnostic is displayed.